tools/ci/src/ci_docs.rs

```diff
@@ -0,0 +1,68 @@
+use clap::{Command, CommandFactory};
+
+use crate::Cli;
+
+pub fn generate_cli_docs() -> String {
```

Comment:
nit: any chance we could use `clap_markdown` like we do for the SpacetimeDB CLI docs?